### DISCUSSION TITLE
ImageMagick overlay HTML color code

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#2278] [Help]
I have a significant number of photos where I need to put a colorful bedspread. I have the HTML color code. I know how to work on files massively, but I don't know how to create overlay with transparency.

Do I have to create a bedspread in PNG?

**Comment by snibgo** [comment]
What is a bedspread? Please show an example of what you want.

**Reply by ghost** [reply]
I present an example. I have an input file. He defines 88% transparency for it and the color is # 2d3545
![test](https://user-images.githubusercontent.com/5912760/87716941-e7c6ef80-c7af-11ea-90aa-6f9b7a51bcf8.jpg)

**Comment by snibgo** [comment]
I suppose you mean 88% opacity, which is 12% transparency.

What version of IM, on what platform? I assume v7 on Windows CMD (command line).

Does this do what you want?
```
magick bedspread.jpg ( +clone -fill #2d3545 -colorize 100 -alpha set -channel A -evaluate set 88% +channel ) -composite out.png
```
Or, more simply:
```
magick bedspread.jpg -fill #2d3545 -colorize 88 d.png
```

**Reply by ghost** [reply]
I have Arch Linux
Version: ImageMagick 7.0.10-24 Q16 x86_64 2020-07-18
I think that this will be my command:
magick sarasa-w-basenie.jpg \( +clone -fill '#2d3545' -colorize 100 -alpha set -channel A -evaluate set 88% +channel \) -composite sasarasa-w-basenie-1.jpg
